src/redux/user/user.actions.ts

```diff
@@ -18,3 +13,12 @@ export const createUser = createAsyncThunk(
     return res.data;
   }
 );
+
+export const fetchUser = createAsyncThunk(
+  "users/getUser",
+  async (id: string) => {
+    console.log("test get user, id: ", id);
```

Comment:
Remove the console log. Not sure if you're purposely leaving some of these as some are deleted and some aren't